components/ShopHeader/index.js

```diff
@@ -14,9 +14,14 @@ export default function ShopHeader({ cartTotal }) {
   return (
     <Styled.ShopHeader>
       <Styled.ShopH1>Shop Name</Styled.ShopH1>
-      <Styled.ShoppingCartLink href={"/ShoppingCart"}>
+      <Styled.ShoppingCartLink
+        aria-label="Checkout link"
+        href={"/ShoppingCart"}
+      >
         <SmallCart />
-        <span>{cartTotal}</span>
+        {cartTotal ? (
+          <Styled.ShoppingCartCounter>{cartTotal}</Styled.ShoppingCartCounter>
+        ) : null}
```

Comment:
even shorter and easier to read:

```suggestion
        {cartTotal && <Styled.ShoppingCartCounter>{cartTotal}</Styled.ShoppingCartCounter>}
```

Reply:
-[x] Good suggestion.